### DISCUSSION TITLE
Question about the consign function in the rgb-std/persistence/inventory.rs

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#139] [General]
Hi there, I found part of the structure of the consignment as follows:
<img width="558" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/1b3c06d4-a994-44df-91da-eba94db54a56">
For the source code in the inventory.rs, line 626-628:
<img width="825" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/bd3e2447-1370-4be6-866f-44a8ba5bbd9c">
Then in line 693 and line 695, there are cosignment.bundiles and the consignment.terminals to store the bundles and terminals which are defined in line 626 and line 628,
<img width="821" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/ecf5fa96-bcc6-4a8b-8442-6d8a74465d79">

 but there is no structure(like the consignment.transitions) to store the transition which assigned the value in line 673:
<img width="576" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/d8bfff48-e82c-460a-a717-510697eb0d71">
So, are there any roadmaps for how to implement the consignment.transitions? (Or maybe I misunderstand the code?)

**Comment by dr-orlovsky** [comment]
Transitions exist only as a part of bundle (which full name is "transition bundle"). There is nothing to implement in this regard.
